### PR TITLE
Send an order confirmation e-mail upon successfull Vipps payment

### LINKED ISSUE #401: Send an e-mail confirmation upon successfull Vipps payment
When an order is captured, the user should receive an email with a confirmation of the purchase.

### PR DESCRIPTION
**Pull request by MorganFH**
## Proposed Changes

- Send an order confirmation email when an order is captured. The email currently looks like this:
image

## Types of Changes

- [x] New feature
- [ ] Bug fix
- [ ] Enhancement/optimization
- [ ] Refactor
- [ ] Style
- [ ] Build/dependencies
- [ ] Other

## Issue(s) fixed or closed by this PR

- closes #401 

## Checklist

- [ ] I have added tests to cover my changes (for features/bugs)
- [x] I am pleased with the readability of the code (if not, state where you need input)
- [x] I have tested the changes and verified that they work and do not break anything (to the best of my ability)

### CODE REVIEW

File: backend/apps/ecommerce/signals.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    An order is captured when the payment status is changed from RESERVED to CAPTURED
+    """
+    # Check that we are updating an order, not creating
+    if not instance._state.adding:
+        previous: Order = sender.objects.get(id=instance.id)
+        if (
+            previous.payment_status == Order.PaymentStatus.RESERVED
+            and instance.payment_status == Order.PaymentStatus.CAPTURED
+        ):
+            send_order_confirmation_mail(instance)
```
Comment (larwaa) [inline backend/apps/ecommerce/signals.py:21]: Could also just wrap this in a try-except block. No strong opinion on this either way.
```suggestion
    try:
        previous: Order = sender.objects.get(id=instance.id)
        if (
            previous.payment_status == Order.PaymentStatus.RESERVED
            and instance.payment_status == Order.PaymentStatus.CAPTURED
        ):
            send_order_confirmation_mail(instance)
    except Order.DoesNotExist:
         continue
```

File: backend/templates/ecommerce/order_confirmation.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      <p>Med vennlig hilsen,</p>
+      <p>
+        Rubberdøk <br />
+        <a href="mailto:kontakt@rubberdok.no"></a>kontakt@rubberdok.no <br /><a
+          href="https://indokntnu.no"
+          >indokntnu.no</a
+        >
+      </p>
+      <img
+        src="https://raw.githubusercontent.com/rubberdok/indok-web/docs/assets/rubberdok_logo_signature.png"
```
Comment (larwaa) [inline backend/templates/ecommerce/order_confirmation.html:79]:
```suggestion
        src="https://indokweb-assets.s3.eu-north-1.amazonaws.com/mail/logo_signature.png"
```

**Review by larwaa — CHANGES_REQUESTED** [review]
Looks very good, minor stuff.

File: backend/apps/ecommerce/signals.py (hunk 1 above)
Comment (MorganFH) [reply backend/apps/ecommerce/signals.py:21]: Agreed, one less query to the DB with the first option 🤷‍♂️

**Review by larwaa — APPROVED** [review]
LGTM 🌟

**Pull request merged**